code/__DEFINES/clockcult.dm

```diff
@@ -33,3 +33,6 @@
 //Name types
 #define CLOCKCULT_MASTER 1
 #define CLOCKCULT_RECRUIT 0
+
+//Heralds beacon activation time
+#define HERALDS_BEACON_TIME 30 * 60 * 10
```

Comment:
Why use a mathematical operation for this? Just use 18,000 :P

Reply:
cause I didnt know what 30 * 60 * 10 is

Reply:
```suggestion
#define HERALDS_BEACON_TIME 18000
```